docs/UserGuide.adoc

```diff
@@ -506,11 +506,54 @@ The `redo` command fails as there are no `undo` commands executed previously.
 `redo` (reapplies the `clear` command) +
 // end::undoredo[]
 
+// tag::setpath[]
+=== Setting the path to local profile pictures folder : `setpath`
+
+Sets the folder location to access and import debtors' profile pictures. +
+Paths should be absolute paths and not relative ones. Also, paths should point to the folder rather than the picture itself. +
+
+It is recommended to create a folder and fill it up with the necessary profile pictures for the application. If there is a need to add a new picture, it should be added into this specific folder. +
+
+The folder should only contain pictures with the `.jpg` extension. Moreover the pictures should be named after the debtors in the application's database. +
+
+For example, for the debtor Alex Yeoh, the name of his picture should be `AlexYeoh.jpg` +
```

Comment:
her* since our ui pics show a lady's pic whenn Alex is selected

Reply:
noted